Dockerfile

```diff
@@ -114,6 +114,9 @@ WORKDIR /app/packages/cardano-services
 CMD ["node", "dist/cjs/cli.js", "start-blockfrost-worker"]
 
 FROM cardano-services as pg-boss-worker
+WORKDIR /config
+COPY compose/schedules.json .
```

Comment:
Does this mean that the image must be rebuilt in case the `schedules.json` file changes?
Previosly it was a mapped volume, so any change would apply with a container restart.

Reply:
Resolving because updating `schedules.json` is not meat to be a frequest task (see comments below).